libs/fairness/src/lib/util/ErrorMetrics.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { localization } from "@responsible-ai/localization";
+
+export interface IErrorOption {
+  key: string;
+  title: string;
+  isMinimization: boolean;
+  description?: string;
+}
+
+export const errorOptions: { [key: string]: IErrorOption } = {
+  disabled: {
+    description: localization.Fairness.Metrics.errorMetricDisabledDescription,
+    isMinimization: false,
+    key: "disabled",
+    title: localization.Fairness.Metrics.errorMetricDisabled
+  },
+  enabled: {
+    description: localization.Fairness.Metrics.errorMetricEnabledDescription,
```

Comment:
Are all these properties getting used somewhere? I can't seem to find them in the code. If not, we should probably not have this and just use boolean values for enabled/disabled.